css/style.css

```diff
@@ -36,14 +36,40 @@ p {
 }
 
 .points {
-    list-style: none;
     font-size: 1.25em;
 }
 
+ul.points {
+    list-style: none;
+}
+
 ol {
   font-size: 1.2em;
 }
 
+ol.points {
+    list-style: none;
+    counter-reset: step;
+}
+
+ol.points > li {
+margin-bottom: 10px;
+}
+
+
+ol.points > li::before {
+    background: #1e2b4c;
+    color: #fff;
+    display: inline-block;
+    content: counter(step);
+    counter-increment: step;
+    width: 45px;
+    height: 45px;
+    border-radius: 23px;
+    text-align: center;
+    margin-right: 10px;
+}
```

Comment:
Not sure if this is really necessary but thought I'd bring it to your attention. This custom number for the list items cause the text to be in line with the number. When text goes to a new line it's under the number not under the text like it normally is. Showcase: 
Custom numbers: 
<img width="524" alt="Screen Shot 2022-10-03 at 5 14 42 PM" src="https://user-images.githubusercontent.com/25210542/193708227-fc6e4998-0e17-4b32-8d23-34b94f3b49bc.png">
Traditional numbers:
<img width="490" alt="Screen Shot 2022-10-03 at 5 15 20 PM" src="https://user-images.githubusercontent.com/25210542/193708254-5227d0d4-ab45-4d26-b816-0d2bd36e797e.png">

Reply:
I also feel like the numbers are a little large in comparison to the text.

Reply:
<img width="1015" alt="Screen Shot 2022-10-04 at 11 13 30 AM" src="https://user-images.githubusercontent.com/10934485/193895094-ecf68b72-5f4f-4a17-aa31-1fe39fe7196a.png">